pipelines/rbsa/tasks/group_sites.py

```diff
@@ -74,6 +79,7 @@ def _task(self):
 
         self.validate(area_loads)
         self.on_complete({self.output_artifact_area_load: area_loads})
+        self.on_complete({self.output_artifact_full_zipcodes: full_zipcodes}) 
```

Comment:
I know this is still in DRAFT, but this can all be made in one call, like:
```
self.on_complete({self.output_artifact_area_load: area_loads, self.output_artifact_full_zipcodes: full_zipcodes}) 
```

Reply:
fixed